src/relay/backend/contrib/ethosn/capabilities.h

```diff
@@ -38,13 +39,14 @@ namespace relay {
 namespace contrib {
 namespace ethosn {
 
-/* Ethos-N variants (N77, N57 and N37)
+/* Ethos-N variants (N77, N57, N37 and N78)
```

Comment:
Please fix this to have the proper names, Ethos-N77, Ethos-N37, Ethos-N57 and Ethos-N78.

Reply:
Done.